ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -273,6 +275,19 @@ tableSource
     | ID_DATE_SUFFIX
     ;
 
+tableFunction
+    : qualifiedName LT_PRTHS
+        (tableFunctionArgument (COMMA tableFunctionArgument)*)? RT_PRTHS
+    ;
+
+tableFunctionArgument
+    : (ident EQUAL)? tableFunctionArgValue
```

Comment:
You can specify `tableFunctionArgumentName` and list all of them to have them checked on parsing.

Reply:
Even if we restrict here, analyzer has to again read and verify the argument names. Why multiple times and also create keywords for funciton argument names. Would prefer dynamic arguments and anlayze them.

Also removed this part of code in current revision.